frontend/src/components/questions/columns.tsx

```diff
@@ -62,11 +63,14 @@ export const columns: ColumnDef<Question>[] = [
     id: "actions",
     header: "Actions",
     cell: ({ row }) => {
+      const id = row.id as string;
       return (
         <div className="flex gap-2 justify-between">
+          <Link href={`/questions/${id}/edit`}>
```

Comment:
can all users edit? shouldn't it be only for the questions they posted - maybe we should put this in the user page or as an optional setting